<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-L6NUOVH" name="com.example.testframework.junit.UserServiceTest" tests="1" failures="1" timestamp="4 Aug 2017 15:15:27 GMT" time="0.002" errors="0">
  <testcase name="testLoadSpringBean" time="0.002" classname="com.example.testframework.junit.UserServiceTest">
    <failure type="org.unitils.core.UnitilsException" message="An exception occured during the loading of core module hibernate with module class name org.unitils.orm.hibernate.HibernateModule">
      <![CDATA[org.unitils.core.UnitilsException: An exception occured during the loading of core module hibernate with module class name org.unitils.orm.hibernate.HibernateModule
	at org.unitils.core.ModulesLoader.createAndInitializeModules(ModulesLoader.java:154)
	at org.unitils.core.ModulesLoader.loadModules(ModulesLoader.java:121)
	at org.unitils.core.Unitils.createModulesRepository(Unitils.java:222)
	at org.unitils.core.Unitils.init(Unitils.java:118)
	at org.unitils.core.Unitils.init(Unitils.java:105)
	at org.unitils.core.Unitils.initSingletonInstance(Unitils.java:74)
	at org.unitils.core.Unitils.getInstance(Unitils.java:52)
	at org.unitils.UnitilsTestNG.getUnitils(UnitilsTestNG.java:162)
	at org.unitils.UnitilsTestNG.getTestListener(UnitilsTestNG.java:152)
	at org.unitils.UnitilsTestNG.unitilsBeforeClass(UnitilsTestNG.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.unitils.core.UnitilsException: Could not load class with name org.hibernate.cfg.AnnotationConfiguration
	at org.unitils.util.ReflectionUtils.getClassWithName(ReflectionUtils.java:612)
	at org.unitils.orm.hibernate.HibernateModule.init(HibernateModule.java:91)
	at org.unitils.core.ModulesLoader.createAndInitializeModules(ModulesLoader.java:151)
	... 33 more
Caused by: java.lang.ClassNotFoundException: org.hibernate.cfg.AnnotationConfiguration
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at org.unitils.util.ReflectionUtils.getClassWithName(ReflectionUtils.java:609)
	... 35 more
]]>
    </failure>
  </testcase> <!-- testLoadSpringBean -->
</testsuite> <!-- com.example.testframework.junit.UserServiceTest -->
